src/main/scala/info/bethard/timenorm/formal/Readers.scala

```diff
@@ -1,13 +1,19 @@
 package info.bethard.timenorm.formal
 
-import java.time.{DayOfWeek, Month}
+import java.time.{DayOfWeek, Month, LocalDateTime}
 import java.time.temporal.{IsoFields, WeekFields, ChronoField, ChronoUnit}
 
 import info.bethard.anafora.{Properties, Data, Entity}
 import info.bethard.timenorm.field._
 
 object AnaforaReader {
-  class Exception(message: String) extends java.lang.Exception(message)
+    class Exception(message: String) extends java.lang.Exception(message)
+}
+
+class AnaforaReader(dct: SimpleInterval)(implicit data: Data) {
+
+  val DCT = dct
```

Comment:
Just declare it as `AnaforaReader(val DCT: SimpleInterval)` if you want a publicly accessible `DCT`.